development-vm/README.md

```diff
@@ -1,276 +1,212 @@
+* [Vagrant](https://www.vagrantup.com/downloads.html)
+* vagrant-dns plugin: `vagrant plugin install vagrant-dns`
 
-    ~/govuk/govuk-puppet
+## 2. Create your GitHub accounts
 
-the following commands will bootstrap your VM:
+1. Set up a [GitHub](https://www.github.com) account.
+1. Get yourself added to the [alphagov organisation](https://github.com/alphagov). Your tech lead will be able to facilitate this.
+1. Set up a GitHub Enterprise (private) account and be added to the `gds` organisation.
+1. [Generate and register an SSH key pair](https://help.github.com/categories/56/articles) for your Mac for your GitHub account.
+1. Import the SSH key into your keychain. Once you’ve done this, it’ll be available to the VM you'll install in the next step.
 
-    cd ~/govuk/govuk-puppet/development-vm
-    ./bootstrap.sh
+        mac$ /usr/bin/ssh-add -K yourkey
 
-and follow the instructions. Your machine will be automatically
-provisioned so you shouldn't have to do anything once it's finished.
+1. Test that it all works by running `ssh -T git@github.com`
 
-Once your VM is running you should be able to SSH into it:
+Now check out the gov.uk Puppet repo, which contains the configuration and automation you'll need to get up and running:
 
-    vagrant ssh
+    mac$ mkdir ~/govuk
+    mac$ cd ~/govuk
+    mac$ git clone git@github.com:alphagov/govuk-puppet.git
 
-and that's it. Now you can get to work!
+## 3. Create a user in Integration and CI
 
-## Extras
+User accounts in our integration and CI environments are managed in the [govuk-puppet](https://github.com/alphagov/govuk-puppet) repository that you just cloned.
 
-* Your VM comes pre-configured with an IP address. This is visible in
-  the [Vagrantfile](./Vagrantfile) (but currently defaults to `10.1.1.254`)
-* If you have < 8GB of RAM on your host machine, you will need to reduce the
-  RAM available to the VM. You can also add extra RAM if you require more.
-  You can do both of these things in a `Vagrantfile.localconfig` file in this
-  directory, which is automatically read by Vagrant (don't forget to run
-  `vagrant reload`!):
+To create a new account, start by creating an SSH key of sufficient key strength.
 
-        $ cat ./Vagrantfile.localconfig
-        config.vm.provider :virtualbox do |vm|
-          vm.customize [ "modifyvm", :id, "--memory", "1024", "--cpus", "2" ]
-        end
+    mac$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com" -f ~/.ssh/alphagov
 
-* If you want to be able to SSH into your VM directly, consider using `vagrant
-  ssh` as it'll always do the right thing. If you need direct access (for
-  `rsync`, `scp` or similar), you'll need to manually configure your ssh
-  configuration: run `vagrant ssh-config --host dev` and paste the output into
-  your `~/.ssh/config`. It'll now be available to SSH into using `ssh dev`.
+Now create a user manifest in `~/govuk/govuk-puppet/modules/users/manifests` with your username and the public key we just created.
```

Comment:
Now create a user manifest in `~/govuk/govuk-puppet/modules/users/manifests` with the username and public key you just created.